views/templates/admin/_configure/helpers/form/form.tpl

```diff
@@ -27,26 +27,81 @@
+          <input type="radio" name="{$input.name|escape:'html':'UTF-8'}" id="{$value.id|intval}"
+                 value="{$value.value|escape:'html':'UTF-8'}"
+                 {if $fields_value[$input.name] == $value.value}checked="checked"{/if}
+                  {if isset($input.disabled) && $input.disabled}disabled="disabled"{/if} />
+          <label class="t" for="{$value.id|intval}">
+              {if isset($input.is_bool) && $input.is_bool == true}
+                  {if $value.value == 1}
+                    <img src="../img/admin/enabled.gif" alt="{$value.label}"
+                         title="{$value.label|escape:'html':'UTF-8'}"/>
+                  {else}
+                    <img src="../img/admin/disabled.gif" alt="{$value.label}"
+                         title="{$value.label|escape:'html':'UTF-8'}"/>
+                  {/if}
+              {else}
+                  {$value.label|escape:'html':'UTF-8'}
+              {/if}
+          </label>
+            {if isset($input.br) && $input.br}<br/>{/if}
+            {if isset($value.p) && $value.p}<p>{$value.p}</p>{/if}
+        {/foreach}
+    {elseif $input.type == 'emailalerts_tags'}
+      {literal}
+        <script type="text/javascript">
+          $().ready(function () {
+            var input_id = '{/literal}{if isset($input.id)}{$input.id}{else}{$input.name}{/if}{literal}';
+            $('#' + input_id).tagify({delimiters: [13, 44], addTagPrompt: '{/literal}{l s='Add email'}{literal}'});
+            $({/literal}'#{$table}{literal}_form').submit(function () {
+              $(this).find('#' + input_id).val($('#' + input_id).tagify('serialize'));
+            });
+          });
+        </script>
+      {/literal}
+      {assign var='value_text' value=$fields_value[$input.name]}
+      {if isset($input.maxchar) || isset($input.prefix) || isset($input.suffix)}
+        <div class="input-group{if isset($input.class)} {$input.class}{/if}">
+      {/if}
+          {if isset($input.maxchar) && $input.maxchar}
+            <span id="{if isset($input.id)}{$input.id}{else}{$input.name}{/if}_counter" class="input-group-addon"><span
+                class="text-count-down">{$input.maxchar|intval}</span></span>
+          {/if}
+          {if isset($input.prefix)}
+            <span class="input-group-addon">
+										  {$input.prefix}
+										</span>
+          {/if}
+          <input type="text"
+               name="{$input.name}"
+               id="{if isset($input.id)}{$input.id}{else}{$input.name}{/if}"
+               value="{if isset($input.string_format) && $input.string_format}{$value_text|string_format:$input.string_format|escape:'html':'UTF-8'}{else}{$value_text|escape:'html':'UTF-8'}{/if}"
+               class="{if isset($input.class)}{$input.class}{/if} tagify"
+                {if isset($input.size)} size="{$input.size}"{/if}
+                {if isset($input.maxchar) && $input.maxchar} data-maxchar="{$input.maxchar|intval}"{/if}
```

Comment:
```suggestion
                {if isset($input.maxchar) && $input.maxchar} data-maxchar="{$input.maxchar|intval}"{/if}
```

I'm not sure if some of those settings are relevant for this special field. It's a special field so you control options that it has and accepts 👍

Reply:
Agree, removed all the useless settings 😉